README.md

```diff
@@ -455,6 +455,34 @@ There is also a command shortcut for specifying this type of lifecycle rule (3).
 terra resource create gcs-bucket --name=mybucket --bucket-name=mybucket --auto-delete=365
 ```
 
+##### GCS bucket object reference
+A reference to an GCS bucket object can be created by calling
+```
+terra resource add-ref gcs-object --name=referencename --bucket-name=mybucket --object-name=myobject
+```
+
+**Reference to a file or folder** 
```

Comment:
nit: for this and below, please make it a new section (i.e. start with "#####" and add it to the table of contents at the top).